cPP_DSC.adoc

```diff
@@ -121,7 +121,7 @@ PP Date: {revdate}
 
 === TOE Overview
 
-The Target of Evaluation (TOE) is a Dedicated Security Component (DSC). In the context of this cPP, a DSC is the combination of a hardware component and its controlling OS or firmware. The firmware should be dedicated to providing the encompassing platform with services for the provisioning, protection, and use of Security Data Objects (SDOs), which include keys, identities, attributes, and other types of Security Data Elements (SDEs). See <<RepofTOE>> for an example of a TOE representation.
+The Target of Evaluation (TOE) is a Dedicated Security Component (DSC). In the context of this cPP, a DSC is the combination of one or more hardware component(s) and its controlling OS or firmware. The firmware should be dedicated to providing the encompassing platform with services for the provisioning, protection, and use of Security Data Objects (SDOs), which are composed of Security Data Elements (SDEs) such as keys, identities, attributes. See <<RepofTOE>> for an example of a TOE representation.
```

Comment:
The paragraph below mentions that a DSC can contain multiple hardware components.

The sentence "use of Security Data Objects (SDOs), which include keys, identities, attributes, and other types of Security Data Elements (SDEs)." sounded like the SDOs are the keys/identities/attributes, but as I understand it those are the SDEs. The SDOs are composed of SDEs.

Reply:
I can agree with this change